en/hosting_guide/configuration.rst

```diff
@@ -3,6 +3,224 @@ Server Configuration
 
 To know more about server configuration see `requirements`_.
 
+Apache
+------
+
+To use Open Orchestra, you need to configure the different virtual host in your Apache configuration.
```

Comment:
It's obvious for us, but you should mention that in Open Orchestra a Front app is set on a different host than a Back app